design/object-store.md

```diff
@@ -0,0 +1,141 @@
+# Rook Object Store
+
+## Overview
+
+An object store is a collection of resources and services that work together to serve HTTP requests to PUT and GET objects. Rook will automate the configuration of the Ceph resources and services that are necessary to start and maintain a highly available, durable, and performant object store.
+
+The Ceph object store supports S3 and Swift APIs and a multitude of features such as replication of object stores between different zones. The Rook object store is designed to support all of these features, though will take some time to implement them. We welcome contributions! In the meantime, features that are not yet implemented can can configured by using the [Rook toolbox](/Documentation/toolbox.md) to run the `radosgw-admin` and other tools for advanced configuration.
+
+### Prerequisites
+
+A Rook storage cluster must be configured and running in Kubernetes.
+
+## Object Store Walkthrough
+
+When the storage admin is ready to create an object storage, he will specify his desired configuration settings in a yaml file such as the following `object-store.yaml`. This example is a simple object store with metadata that is replicated across different hosts, and the data is erasure coded across multiple devices in the cluster.
+```yaml
+apiVersion: rook.io/v1alpha1
+kind: ObjectStore
+metadata:
+  name: my-store
+  namespace: rook
```

Comment:
whats the relationship between this namespace and the namespace used by the underlying ceph cluster?

Reply:
they must be the same. I'll add a comment about that. It's the same concept for all the CRDs so i have been assuming that detail.